event_sourcery_esdb/__init__.py

```diff
@@ -19,5 +21,8 @@ def __init__(self, esdb: EventStoreDBClient) -> None:
         self._configure(storage_strategy=ESDBStorageStrategy(self._client))
 
     def with_outbox(self, filterer: OutboxFiltererStrategy = dummy_filterer) -> Self:
+        temporal_outbox = f"outbox-test-{uuid4().hex}"
```

Comment:
Do we actually want to include `-test-` in outbox name? Shouldn't this piece be moved over to tests?

Reply:
Yes!!! I 100% agree. will fix this. For some reason, I was assuming this is in `conftest`.